documentation/developers/README.md

```diff
@@ -11,3 +11,8 @@
 * [RFID Readers](./rfid)
 * [Feature Status](./status.md)
 * [Known Issues](./known-issues.md)
+
+## RPC
+
+* [LibZMQ](./libzmq.md)
+* [CLI Client for RPC](./cli-client.md)
```

Comment:
If i get it right this is not necessarily only for developers.
Should the CI Client in the builders/Readme.md under "Advanced" ?

There is also the page builders/rpc-commands.md

Reply:
You are probably right, builders should have the possibility to utilize everything.